### ISSUE TITLE
Invalid url of image in sample

### ISSUE THREAD

**Issue by pyhedgehog** [bigcompany/hooks#1]
Image url https://hook.io/img/robotcat.png in [examples-javascript-image-resize](https://github.com/bigcompany/hooks/blob/0d0cab2e62e2a3aa157875e7f70c03f5ae208ba1/examples-javascript-image-resize/index.js#L8) is non-existant.

**Comment by Marak** [comment]
Yes. Good catch. I restored the image, but `gm` appears to be broken ( i think for several months now )

A bit busy this week, but I'll try to see if I can get the image resize functionality working again this weekend.
